next.config.js

```diff
@@ -5,6 +5,9 @@
 const nextConfig = {
   reactStrictMode: true,
   trailingSlash: true,
+  images: {
```

Comment:
could you explain me what does this?

Reply:
Yes, I added the image component of nextjs and then I had to add domain of the image, but not is neccesary at moment. Sorry.